frontend/index.js

```diff
@@ -3,10 +3,9 @@ import React from 'react';
 import { Provider } from 'react-redux';
 import store from './app/store/configureStore';
 import App from './app/App';
-import { loadCSSFromURLAsync, loadCSSFromString } from '@airtable/blocks/ui';
-import styles from './app/styles/main.js'
+import { loadCSSFromURLAsync } from '@airtable/blocks/ui';
 loadCSSFromURLAsync('https://maxcdn.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css');
-loadCSSFromString(styles);
```

Comment:
Airtable does not have an API to import styles from css/scss. It can only load styles from a string.

I use https://github.com/saas-developer/airtable-build-styles to go from `scss` to a `js` which consists of all the required `css` as one long string. That file is `main.js` which is imported in line 7.

Reply:
I didn't see the file where you are referencing it. 

Does this not work? https://www.airtable.com/developers/apps/api/UI/utils/loadCSSFromURLAsync

Reply:
Please check this post on Airtable community for how this works - https://community.airtable.com/t/how-can-i-bundle-external-css-files-with-block/26842/10